modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -210,6 +213,42 @@ void testManualRebalanceIfMajorityIsLost() throws Exception {
         assertThat(errors, is(empty()));
     }
 
+    /**
+     * Tests that a situation from the test {@link #testInsertFailsIfMajorityIsLost()} it is possible to recover specified partition using
+     * a disaster recovery API.
+     */
+    @Test
+    @ZoneParams(nodes = 5, replicas = 3, partitions = 2)
+    void testManualRebalanceIfMajorityIsLostSpecifyPartitions() throws Exception {
+        int partId = 0;
+        int secondPartId = 1;
+
+        IgniteImpl node0 = cluster.node(0);
+        Table table = node0.tables().table(TABLE_NAME);
+
+        awaitPrimaryReplica(node0, secondPartId);
+
+        stopNodesInParallel(2, 4);
```

Comment:
This test seems to rely on 'both nodes 2 and 4 will get one replica of both partitions each'. Could you please add a comment about it (or, if possible, assert this)?